front/lib/actions/mcp_internal_actions/servers/jira/types.ts

```diff
@@ -100,65 +100,6 @@ export const JiraSortSchema = z.object({
     .describe(`Sort direction. Must be one of: ${SORT_DIRECTIONS.join(", ")}.`),
 });
 
-// Jira entity schemas - shared field definitions
-export const JiraIssueFieldsSchema = z
-  .object({
-    project: z.object({
-      key: z.string(),
-    }),
-    summary: z.string(),
-    description: z
```

Comment:
I change to `description: ADFDocumentSchema.nullable(),` hence I had to move those after the definition of `ADFDocumentSchema`